### ISSUE TITLE
Versionbot removes the newline at the end of package.json

### ISSUE THREAD

**Issue by Page-** [resin-io-modules/resin-procbots#246]
This means that whenever you perform any action that auto formats package.json the newline gets readded and it shows up as a git diff even if the actual contents don't change

**Comment by lurch** [comment]
Duplicate of https://github.com/resin-io/versionist/issues/87 ;-)

**Comment by hedss** [comment]
Yeah it is, and it's a something I keep getting irritated by, so I'm going to bite the bullet and sort it out soon.

**Comment by lurch** [comment]
I guess this issue can be closed after VersionBot upgrades to versionist v2.11.2

**Comment by hedss** [comment]
Yes
